docs/developer/add-new-source.md

```diff
@@ -25,6 +25,9 @@ Prefixes need to be entered in the following places in the yml:
 ### 2.3. `config/prefixes.csv`
 Add prefixes.
 
+### 2.4. `config/context.json`
```

Comment:
## Related issue(s)
- https://github.com/monarch-initiative/mondolib/issues/8

Previously this issue was just about `prefixes.csv`, but now I've added a bit about `context.json`.